cuda_core/cuda/core/experimental/_utils/driver_cu_result_explanations.py

```diff
@@ -0,0 +1,336 @@
+# Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
+
+# To regenerate the dictionary below, navigate to:
+#     https://docs.nvidia.com/cuda/cuda-driver-api/group__CUDA__TYPES.html#group__CUDA__TYPES
+# (Chrome was used before, but probably it works with other browsers, too.)
+# Search for:
+#     enum CUresult
+# With the mouse, select the entire region with the enum definitions:
+#     CUDA_SUCCESS = 0
+#     ...
+#     CUDA_ERROR_UNKNOWN = 999
+#         This indicates that an unknown internal error has occurred.
+# Paste into a file, e.g. raw.txt
+# curl -O https://raw.githubusercontent.com/rwgk/stuff/master/cuda-python/reformat_cuda_enums_from_web_as_py.py
```

Comment:
Should we check in this file somewhere so we can properly track changes / enhancements to it?

Reply:
I was thinking of maybe adding a top-level "toolshed" directory, cuda-python top level.

I'd want to exclude that directory when building wheels.

There is already another script I'd want to move in there ([bootstrap_local_dev.sh](https://github.com/rwgk/stuff/blob/master/cuda-python/bootstrap_local_dev.sh)). I think it'd be a nice home for random helpers like that. WDYT?

Reply:
I would defer to folks like you and @leofang as far as how you'd want to structure the repo, but I agree in not shipping it as part of any of the packages.

Reply:
I went ahead and added the toolshed directory here: commit c99e8d15caf0ac27428e54a6f971015875505df2

It's easier than I thought: that directory is ignored when we build the wheels under cuda_bindings and cuda_core, at least in my interactive testing.

I convinced myself that ruff doesn't ignore it (great).

I'll rerun the tests, to be sure the directory isn't included in the automatically generated wheels.

Reply:
I inspected the CI-generated whl files:
* cuda_bindings-12.8.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
* cuda_core-0.2.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl

The toolshed directory or reformat_cuda_enums_from_web_as_py.py  file are not included in both.